packages/server-wallet/src/protocols/channel-closer.ts

```diff
@@ -11,6 +11,13 @@ import {Channel} from '../models/channel';
 
 import {Defunder} from './defunder';
 
+export const enum WaitingFor {
+  allAllocationItemsToBeExternalDestination = 'ChannelCloser.allAllocationItemsToBeExternalDestination',
+  theirFinalState = 'ChannelCloser.theirFinalState', // i.e. other participants' final states
+  defunding = 'ChannelCloser.defunding',
+  nothing = '',
```

Comment:
Perhaps `nothing` could probably be defined at a higher level and shared between crankers?

Reply:
Yeah that's a good suggestion. The PR below this one in the stack #3286 has recently been updated to make  `waitingFor` optional. So we can set that field to `undefined` instead of `Enum.nothing` and not have the `nothing` option in the enums at all.